packages/opentelemetry-plugin-redis/package.json

```diff
@@ -8,11 +8,14 @@
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
     "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
```

Comment:
Do you think we should remove `"private": true` to make this package available for the next release?

Reply:
Agreed. I removed it from this plugin. I think we should also remove it from other "ready" plugins (separate PR)